_episodes/08-arguments.md

```diff
@@ -4,7 +4,7 @@ teaching: 10
 exercises: 0
 questions:
 - "How do I specify arguments that don't require input values?"
-- "How do I refer to runtime parameters?"
+- "How do I refer to parameters that are only defined at runtime?"
```

Comment:
I think the original question is about [`$(runtime)` ](https://www.commonwl.org/v1.1/CommandLineTool.html#Runtime_environment)

Reply:
Sure. Is the question not still valid in that case?

Would you prefer 

> How do I make use of parameters that are only defined at runtime?

?